### DISCUSSION TITLE
What it Utility.exe for?

### DISCUSSION THREAD

**Discussion by MikeSW17** [Open-Shell/Open-Shell-Menu#1671] [General]
Hi.
I see a 'Utility.exe' file  in the downloads, what's it for?
TIA Mike

**Comment by ge0rdi** [comment]
Various _utility_/_troubleshooting_ things, like `Open-Shell` logging settings, or manual uninstall.
https://github.com/Open-Shell/Open-Shell-Menu/tree/master/Src/Setup/Utility

**Reply by MikeSW17** [reply]
Tks. Been using OpenShell since ... well Classic-Shell days. I guess I won't be needing Utility.exe untill when/if it gets documentation!

**Reply by ge0rdi** [reply]
> I won't be needing Utility.exe

That's exactly right.
You don't need it unless you'll have some issue and are asked to use it.
